sites/all/modules/reol_loan/reol_loan.module

```diff
@@ -265,3 +265,41 @@ function reol_loan_delete_reservations_ajax($user, $local_ids) {
   }
   ajax_deliver($response);
 }
+
+/**
+ * Implements hook_form_FORMID_alter().
+ *
+ * Add ereol specific admin configuration to user settings form.
+ */
+function reol_loan_form_user_admin_settings_alter(&$form, &$form_state, $form_id) {
+  $form['ereol_user'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Ereol user settings'),
+    '#weight' => 0,
+    '#tree' => FALSE,
+  );
+
+  $form['ereol_user']['ereol_loan_text'] = array(
+    '#type' => 'text_format',
+    '#title' => t('Loan text'),
+    '#format' => 'ding_wysiwyg',
+    '#default_value' => variable_get('ereol_loan_text', '')['value'],
```

Comment:
variable_get('ereol_loan_text', '')['value']will fail if the variable is not set.

Could be changed to variable_get('ereol_loan_text', array('value' => ''))['value']

Reply:
Modified variable get